packages/jaeger-ui/package.json

```diff
@@ -69,6 +68,7 @@
     "lodash": "^4.17.19",
     "logfmt": "^1.2.0",
     "lru-memoize": "^1.1.0",
+    "lucide-react": "^0.279.0",
```

Comment:
why are we adding another module? I though the whole point was to restrict ourselves to react-icons.